<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Winner</class>
 <widget class="QWidget" name="Winner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CONGRATULATIONS!</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../imgresources.qrc">
    <normaloff>:/img/pacman/1.png</normaloff>:/img/pacman/1.png</iconset>
  </property>
  <widget class="QLabel" name="background_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>448</width>
     <height>600</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/img/background);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="logo_label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>331</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/img/pacman_logo);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="congrats_label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>150</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>CONGRATULATIONS</string>
   </property>
  </widget>
  <widget class="QLabel" name="amazing_label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>190</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>YOU ARE AMAZING!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="moreInfo_button">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>520</y>
     <width>88</width>
     <height>33</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 0, 128);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>MORE INFO</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exit_button">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>520</y>
     <width>88</width>
     <height>33</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 0, 128);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>EXIT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mainMenu_button">
   <property name="geometry">
    <rect>
     <x>177</x>
     <y>520</y>
     <width>88</width>
     <height>33</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 0, 128);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>MAIN MENU</string>
   </property>
  </widget>
  <widget class="QLabel" name="gifwin_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>240</y>
     <width>351</width>
     <height>251</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../imgresources.qrc"/>
 </resources>
 <connections/>
</ui>
